en/uavcan/developer.md

```diff
@@ -1,29 +1,7 @@
 # UAVCAN Development
 
-This topic/section has information that is relevant to developers who want to add support for new [UAVCAN](http://uavcan.org) hardware to the PX4 Autopilot.
-
-:::note
-[Hardware > UAVCAN Peripherals](../uavcan/README.md) contains information about using existing/supported UAVCAN components with PX4.
-:::
-
-## Upgrading Node Firmware
-
-The PX4 middleware will automatically upgrade firmware on UAVCAN nodes if the matching firmware is supplied.
-The process and requirements are described on the [UAVCAN Firmware](../uavcan/node_firmware.md) page.
-
 ### Debugging with Zubax Babel
```

Comment:
This is the only topic left in UAVCAN development. Is it/can it/should it go somewhere else? 

Or to put it another way, what happened to the "development story"

Reply:
Ah, thanks for catching this, I didn't take a close enough look at the diff. The reason I'm not migrating the Zubax Babel information right now is because I'd like to revisit it in 2-4 weeks pending some research/testing on the state of the dronecan gui tool and also implementing a mavlink slcan tunnel into PX4. I'll be pulling this out into a full article on CAN development.

If you'd prefer, I can write a stub for that article now with this in it.

Reply:
@coderkalyan Thanks. 

Yes

1. The documentation should always be complete and ready to publish. So yes, we need to clean this.
2. We shouldn't lose information "accidentally"
3. All links to old docs still end up at sensible end points.

If you can create a reasonable stub article then:
- Create new doc for DroneCan or Can development that includes this section, and any other links and information you can find quickly. Perhaps a note that this section is being reconstructed. I would be happy to have links to firmware updates etc as well. 
- Redirect the old doc to the new doc.
- Fix up links to the new docs  (even with the redirects, this is better)